apps/web/src/views/Swap/V3Swap/types.ts

```diff
@@ -0,0 +1,16 @@
+export enum ConfirmModalState {
+  REVIEWING,
+  RESETTING_USDT,
```

Comment:
hmm I was not expecting to fix that in this pull request. But anyway let's not name `USDT` explicitly. Can name it `RESETTING_APPROVAL` as we don't know if there're other tokens have similar restriction